easybuild/easyblocks/c/cuda.py

```diff
@@ -356,10 +356,18 @@ def make_module_req_guess(self):
             lib_path.append(os.path.join('nvvm', 'lib64'))
             inc_path.append(os.path.join('nvvm', 'include'))
 
+        library_path = ['lib64', os.path.join('stubs', 'lib64')]
```

Comment:
You don't want the `stubs` in the final RPATH, these need to come from the OS at runtime

Reply:
Ah, sorry, this is explicitly filtered in framework (I am pretty sure...)

Reply:
I think this can be much simpler than this, there's no need for the checking, you can just directly set
```
'LIBRARY_PATH': lib_path + [os.path.join('stubs', 'lib64')],
```
below. The stubs location is filtered out by the rpath wrappers at link time (which is correct).

Reply:
The only difference between the link libraries and runtime libraries is the stubs location

Reply:
If we can always add `extras/CUPTI/lib64` and `nvvm/lib64` to `LIBRARY_PATH`, then we can implement it as you suggested.

BTW, while it may not relevant anymore, `CUPTI` and `nvvm` lib dirs should only be added for CUDA 7 or newer. That's currently not the case in the PR, but would be following your suggestion.

Reply:
The directories being added are just guesses, if they don't exist they won't be included in the final module file (I think...)

Reply:
Implemented suggestions in https://github.com/easybuilders/easybuild-easyblocks/pull/3516/commits/5077104baf2e0c80d035ba65e5bca3e4d7ce30d1